robot/Cumulus/tests/browser/recurring_donations/zcreate_open_erd_record.robot

```diff
@@ -3,14 +3,14 @@ Resource        robot/Cumulus/resources/NPSP.robot
 Library         cumulusci.robotframework.PageObjects
 ...             robot/Cumulus/resources/NPSPSettingsPageObject.py
 ...             robot/Cumulus/resources/ContactPageObject.py
-...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/RecurringDonationsPageObject.pyS
```

Comment:
.pyS ? I think this was done unintentionally. Can you fix ?

Reply:
Thanks, fixed the typo